### ISSUE TITLE
Enable bwa metagenomics by default

### ISSUE THREAD

**Issue by yesimon** [dnanexus/viral-ngs#53]
The newly introduced bwa metagenomics step aligns against an index that requires a flat 8G memory. Runtime should be on par with a typical human bwa alignment. Due to the relatively high runtime we need to consider the instance types being targeted and whether it will be cost-effective.

**Comment by mlin** [comment]
How should this fit in to (alongside?) the existing kraken / krona workflow?
Where can we get the index?

**Comment by dpark01** [comment]
Yeah I think there's some more work to be done here on our end first--there's some validation work on the method itself, but also we'll want to write out the workflow and provide an equivalent db. Maybe after @tomkinsc gets a few more bits converted to WDL, we can propose a WDL for bwa-lca metagenomics as well. @yesimon when you say high runtime, how long are we talking (on, say, a c4.2xlarge or c3.2xlarge)?

**Comment by mlin** [comment]
@dpark01 Got it. Where's the DIAMOND part relatively?
